src/miner.cpp

```diff
@@ -238,7 +240,19 @@ static void AddTokenRestartTxs(BlockContext &blockCtx,
 
     const auto tokensLocked = mnview.AreTokensLocked(loanTokenIds);
 
-    if (!tokenPricesValid || tokensLocked) {
+    bool poolDisabled{false};
+    mnview.ForEachPoolPair([&](DCT_ID const &poolId, const CPoolPair &pool) {
+        if (loanTokenIds.count(pool.idTokenA.v) || loanTokenIds.count(pool.idTokenB.v) ||
+            collateralTokenIds.count(pool.idTokenA.v) || collateralTokenIds.count(pool.idTokenB.v)) {
+            if (!pool.status) {
```

Comment:
does this lead to issue with old dToken/v1-DUSD pools? cause they all include DUSD, so come up in this list, but its fine that they are disabled.
only dToken-DUSD pools, gateways and collateral-DFI pools are "needed"